lib/routes/sustainabilitymag/articles.ts

```diff
@@ -41,6 +41,8 @@ const render = (widgets) =>
             switch (w.type) {
                 case 'text':
                     return w.html;
+                case 'blockquote':
+                    return w.html;
```

Comment:
```suggestion
                    return `<blockquote>${w.html}</blockquote>`;
```

Reply:
Oups nice catch! Thanks for the suggestion.